app/views/users/_form.html.erb

```diff
@@ -33,7 +33,7 @@
   <%= hidden_field_tag(:from_cart, false) %>
   <%= hidden_field_tag(:partial_to_render, 'form') %>
 
-  <%= render partial: 'terms_of_service/user_accepted', locals: {f: f} unless action_name == 'new' %>
+  <%= render partial: 'terms_of_service/user_accepted', locals: {f: f} if (current_user == @user || !current_user) %>
```

Comment:
Hmm, you're right. Admins may want to know if a user has accepted the ToS. Making quick change now